sceptre/cli/launch.py

```diff
@@ -1,28 +1,43 @@
+import logging
+from typing import List
+
 import click
+from click import Context
+from colorama import Fore, Style
 
+from sceptre.cli.helpers import catch_exceptions, confirmation, stack_status_exit_code
 from sceptre.context import SceptreContext
-from sceptre.cli.helpers import catch_exceptions
-from sceptre.cli.helpers import confirmation
-from sceptre.cli.helpers import stack_status_exit_code
+from sceptre.exceptions import DependencyDoesNotExistError
 from sceptre.plan.plan import SceptrePlan
+from sceptre.stack import Stack
+from sceptre.stack_status import StackStatus
+
+logger = logging.getLogger(__name__)
 
 
 @click.command(name="launch", short_help="Launch a Stack or StackGroup.")
 @click.argument("path")
+@click.option("-y", "--yes", is_flag=True, help="Assume yes to all questions.")
 @click.option(
-    "-y", "--yes", is_flag=True, help="Assume yes to all questions."
+    "-p",
+    "--prune",
+    is_flag=True,
+    help="If set, will delete all stacks in the command path marked as obsolete."
 )
 @click.pass_context
 @catch_exceptions
-def launch_command(ctx, path, yes):
+def launch_command(ctx: Context, path: str, yes: bool, prune: bool):
     """
-    Launch a Stack or StackGroup for a given config PATH.
-    \f
+    Launch a Stack or StackGroup for a given config PATH. This command is intended as a catch-all
+    command that will apply any changes from Stack Configs indicated via the path.
 
-    :param path: The path to launch. Can be a Stack or StackGroup.
-    :type path: str
-    :param yes: A flag to answer 'yes' to all CLI questions.
-    :type yes: bool
```

Comment:
These aren't necessary on CLI commands and they end up just printing redundant garbage in the docs. We should pull these out from all the commands.